frontend/src/actionCreators/usersActionCreator.tsx

```diff
@@ -30,7 +30,7 @@ export const getUsersAction = (params: API.IPaginateParams) => (dispatch: Functi
     .post(ENVIRONMENT.apiUrl + API.POST_USERS(), params)
     .then((response: AxiosResponse) => {
       dispatch(success(actionTypes.GET_USERS, response.status));
-      dispatch(adminActions.storeUsers(response.data));
+      dispatch(adminActions.storeUsers({ ...response.data, page: response.data.page - 1 }));
```

Comment:
Given that the frontend "page" differs in value from what the server sends (i.e page starts at "0" in the frontend, and at "1" in the backend") - I would suggest we use a different name in the frontend to avoid confusion. 

I faced this in my table code as well and went with `pageIndex = page - 1`

In my opinion this avoids accidental `page = page` thinking which would cause bugs. What do you think?

Reply:
I agree, make sense to me